src/libguac/guacamole/display.h

```diff
@@ -254,6 +254,13 @@ void guac_display_notify_user_left(guac_display* display, guac_user* user);
  * mouse button. This function automatically invokes
  * guac_display_end_mouse_frame().
  *
+ * NOTE: If using guac_display_render_thread, the
+ * guac_display_render_thread_notify_user_moved_mouse() function instead. If
```

Comment:
I think this part of the comment is missing a couple of words - maybe `use the...function, instead`, or `the... function should be used, instead.` ??

Reply:
Oops, yes.